src/security/run.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+solutionDirectoryPath=''
+solutionRelativePath=''
+outputDirectoryPath=''
+
+while test $# -gt 0; do
+  case "$1" in
+    -solutionDirectoryPath)
+      shift
+      solutionDirectoryPath=$1
+      shift
+      ;;
+    -outputDirectoryPath)
+      shift
+      outputDirectoryPath=$1
+      shift
+      ;;
+    -solutionRelativePath)
+      shift
+      solutionRelativePath=$1
+      shift
+      ;;
+  *)
+      break
+      ;;
+  esac
+done
+
+if ! docker version; then
+  exit
+fi
+
+echo "Docker pulling image godeltech/codereview.orchestrator..."
+
+if ! docker image pull godeltech/codereview.orchestrator; then
+  exit
+fi
+
+tmpfile=$(mktemp)
+scriptRoot=$(dirname "$0")
+
+awk "{sub(\"{SOLUTION_FILE_PATH}\",\"$solutionRelativePath\")}1" "$scriptRoot/workflow.yaml" > "$tmpfile"
```

Comment:
you can remove this line. It was used to replace the `{SOLUTION_FILE_PATH}` with the relative solution path in the workflow file.

Reply:
Also, you can remove `SolutionRelativePath` from the parameters of the script as well as from the README file

Reply:
Done